CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 * [BUGFIX] Ingester: If push request contained both valid and invalid samples, valid samples were ingested but not stored to WAL of the chunks storage. This has been fixed. #3067
 * [BUGFIX] Ruler: Config API would return both the `record` and `alert` in `YAML` response keys even when one of them must be empty. #3120
 * [BUGFIX] Index page now uses configured HTTP path prefix when creating links. #3126
+* [BUGFIX] Configs: prevent validation of templates to fail when using template functions. #2490
```

Comment:
We typically add PR numbers to the changelog.

```suggestion
* [BUGFIX] Configs: prevent validation of templates to fail when using template functions. #3157
```

Reply:
Hi @pstibrany I added a test and fixed the PR number in the changelog

Reply:
Thank you. Feel free to merge once CI passes.

Reply:
yeah no worries, I don't have write access anyway to can't merge :)